docs/guide/bundles.md

```diff
@@ -29,6 +29,10 @@ see the [How to Remove a Bundle](http://symfony.com/doc/5.0/bundles/remove.html)
 
 ## Core packages
 
+!!! tip
+
+    The core packages are included in Ibexa Open Source.
```

Comment:
Howabout using the active voice?
"Ibexa Open Source includes the core packages."?

Reply:
I've tried to make it clear that core packages are the *only* ones in OSS, without actually saying "only" :)

Reply:
In this case, I like "Ibexa Open Source is composed of the core packages" best, despite its passivevoiciness